pkg/reconciler/app/reconciler.go

```diff
@@ -247,3 +250,59 @@ func (r *Reconciler) updateStatus(desired *v1alpha1.App) (*v1alpha1.App, error)
 
 	return r.KfClientSet.KfV1alpha1().Apps(existing.GetNamespace()).UpdateStatus(existing)
 }
+
+// gcRevisions is necessary because Knative won't scale down revisions
```

Comment:
Can you update the header to say what it does and then why it's necessary?

Reply:
Done